### PR TITLE
Hotels: show error when dates are wrong

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: packages/localization/src/vocabularies/Hotels.js [hunk 1]
```diff
@@ -17,6 +17,8 @@ export default {
   'hotels_search.all_hotels.please_search':
     'Please search for your desired destination.',
   'hotels_search.all_hotels_search_list.no_hotels_found': 'No hotels found.',
+  'hotels_search.all_hotels_search.date_error':
+    'Ops, there seems to be something wrong with the chosen dates',
```
Comment (RobinCsl) [inline packages/localization/src/vocabularies/Hotels.js]: O**o**ps?

Comment (tbergq) [reply packages/localization/src/vocabularies/Hotels.js]: Yeah, you are right, I had to look that up, I did not know. Hopefully, this error should never be shown in production. 🤞

Comment (RobinCsl) [reply packages/localization/src/vocabularies/Hotels.js]: 🤞

**Comment by tbergq** [comment]
@RobinCsl 🚀

**Pull request merged**